GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -2560,7 +2561,7 @@ private void OpenFilesWithDiffTool(IEnumerable<GitItemStatus> items, string firs
                 string output = Module.OpenWithDifftool(item.Name, null, firstRevision, secondRevision, "", item.IsTracked);
                 if (!string.IsNullOrEmpty(output))
                 {
-                    MessageBox.Show(this, output);
+                    MessageBox.Show(this, output, "Information", MessageBoxButtons.OK, MessageBoxIcon.Information);
```

Comment:
I think this should be an error:
https://github.com/gitextensions/gitextensions/blob/master/GitCommands/Git/GitModule.cs#L3561-L3563

Reply:
Right. I should have open Module.OpenWithDifftool.